src/messaging/ExchangeMgr.cpp

```diff
@@ -306,6 +310,12 @@ void ExchangeManager::OnMessageReceived(const PacketHeader & packetHeader, const
             // Using same error message for all errors to reduce code size.
             ChipLogError(ExchangeManager, "OnMessageReceived failed, err = %s", ErrorStr(err));
         }
+
+        // close Exchange for group message
```

Comment:
This should have happened automatically in the exchange code, afaict, so this is double-closing.

Reply:
removed